crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -73,6 +73,47 @@ mod ffi {
         AbsoluteSpacing,
     }
 
+    /// The style strategy tells the font matching algorithm what type of fonts should
+    /// be used to find an appropriate default family.
+    #[repr(i32)]
+    #[namespace = "rust::cxxqtlib1"]
+    #[derive(Debug)]
+    enum QFontStyleStrategy {
+        /// the default style strategy. It does not prefer any type of font
+        PreferDefault = 0x0001,
+        /// prefers bitmap fonts (as opposed to outline fonts).
+        PreferBitmap = 0x0002,
+        /// prefers device fonts.
+        PreferDevice = 0x0004,
+        /// prefers outline fonts (as opposed to bitmap fonts).
+        PreferOutline = 0x0008,
+        /// forces the use of outline fonts.
+        ForceOutline = 0x0010,
+        /// don't antialias the fonts.
+        PreferMatch = 0x0020,
+        /// avoid subpixel antialiasing on the fonts if possible
+        PreferQuality = 0x0040,
```

Comment:
The Qt docs for PreferMatch and PreferQuality suggest that these are flags that may be `OR`-ed with the other values of this enum.

With this wrapping, they are only available themselves, not as an `OR`-ed combination of the other flags.

Are they still valid StyleStrategy values without `OR`ing them together?

Reply:
StyleStrategy is still valid without them.
I will comment them until we will support "OR"

Reply:
Please use `//`, not `///` for commenting-out the PreferMatch, and PreferQuality.
Otherwise they end up as doc comments on PreferAntialias :sweat_smile:

Reply:
oops :)